src/tests/test-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { MemoryRouter } from "react-router-dom";
+import { render } from "@testing-library/react";
+import { ReactElement } from "react";
+
+// -----------------------------------------------------------------------------------------
+// #region Functions
+// -----------------------------------------------------------------------------------------
+
+/**
+ * Returns a React component wrapped in a MemoryRouter. Used for Components that
+ * must be rendered inside a Route Component.
+ *
+ * @param component
+ * @returns {ReactComponent}
+ */
+
+const _renderWithRouter = (ui: ReactElement, route: string = "/") => {
```

Comment:
@SaidShah Sorry for the back & forth - thanks for adding these types. Can you update the JSDoc comment to match? (Whether that's renaming the param or updating the comment - `ui` to me is a bit vague, `component` seems more appropriate)